charts/chaostoolkit/experiments.yaml

```diff
@@ -6,7 +6,7 @@ description:
     Deletes a pod belonging to a deployment/statefulset/daemonset
 kind: ChaosExperiment
 metadata:
-  name: k8-pod-delete
+  name: k8s-pod-delete
```

Comment:
it is k8_pod_delete -->  they have a convention being used that way for other exp as well

Reply:
okay!!